File: Programmers/Level1/220815/java/ReverseNatureNumber.java
public class ReverseNatureNumber {
  public int[] solution(long number) {
    int numberLength = getNumberLength(number);
    int[] answer = new int[numberLength];

    for(int i = 0; i < numberLength; i += 1) {
      int remainder = (int) (number % 10);
      int quotient = (int) (number / 10);

      answer[i] = remainder;

      number = quotient;
    }

    return answer;
  }

  public int getNumberLength(long number) {
    return (int)(Math.log10(number)+1);
  }
}

File: Programmers/Level1/220815/javascript/ReverseNatureNumber.js
function solution(number) {
  const stringNumber = `${number}`;
  const answer = new Array(stringNumber.length);

  for (let i = 0; i < stringNumber.length; i += 1) {
    answer[i] = parseInt(stringNumber.charAt(i));
  }

  return answer.reverse();
}
